Documentation/ceph-cluster-crd.md

```diff
@@ -43,6 +43,7 @@ spec:
 
 ## PVC-based Cluster
 
+**NOTE** Kubernetes version should be 1.13.0 or greater to provision OSD on PVC.
```

Comment:
suggestion: 
`**NOTE** Kubernetes version 1.13.0 or greater is required to provision OSDs on PVCs.`

Reply:
updated.